src/test_order.py

```diff
@@ -25,39 +26,33 @@ def test_sign(self):
         buy_amount = 1000000
         kind = "buy"
-        fee_amount = 1
+        fee_amount = 887
 
         pk = "0x4f3edf983ac636a65a842ce7c78d9aa706d3b113bce9c46f30d7d21715b23b1d"
         receiver = Account.from_key(pk).address
-        # Must specify validTom because it fixes the order parameters and
+        # Must specify validTo because it fixes the order parameters and
         # makes the signature deterministic.
-        order = Order(sell_token, buy_token, sell_amount, buy_amount, fee_amount, kind,
-                      receiver, validTo=1234)
-        self.assertEqual(order.signature, "")
-        order.sign("0x4f3edf983ac636a65a842ce7c78d9aa706d3b113bce9c46f30d7d21715b23b1d")
-        self.assertEqual(
-            "0x17f0bf89c33f5a8ebb750904783421ca127ac59c2680672a69aebc08cd0b644562b19c0a97e34a69e26345459cfccce8276410b933eeded485e2d552ccfaa5211b",
-            order.signature
-        )
-
-
-# expected = json.load("""
-# {
-#     "sellToken": "0xDDAfbb505ad214D7b80b1f830fcCc89B60fb7A83",
-#     "buyToken": "0x4ecaba5870353805a9f068101a40e0f32ed605c6",
-#     "receiver": "0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1",
-#     "sellAmount": "1000000",
-#     "buyAmount": "1000000",
-#     "validTo": 1638052668,
-#     "appData": "0x000000000000000000000000000000000000000000000000000000000000ca1f",
-#     "feeAmount": "885",
-#     "kind": "buy",
-#     "partiallyFillable": false,
-#     "sellTokenBalance": "erc20",
-#     "buyTokenBalance": "erc20",
-#     "signingScheme": "eip712",
-#     "signature": "0xf659927f204c1da68526cf209c1523167a2aeb1cbbc2854318d2e7dfb9e8270e29153778afdb605b079af1e5f3cb575ff65b4afdf6546d657df60acf71a1aa5a1c"
-# }""")
+        order = Order(sell_token, buy_token, receiver, sell_amount, buy_amount, fee_amount, kind,
+                      validTo=1639052668)
+        required_signature = "0x547b936d1aac1b0153776887ee23a92097d3988c433b0d9642014c9861e07f8f3e16cbc7d1c2d42d3f3b32b332f2aafc86daace610a0a5968b93733c79c87a1a1b"
+        self.assertEqual(order.sign, required_signature)
+        expected_order = json.load("""{
+            "sellToken": "0xDDAfbb505ad214D7b80b1f830fcCc89B60fb7A83", 
+            "buyToken": "0x4ecaba5870353805a9f068101a40e0f32ed605c6", 
+            "receiver": "0xD2Cc479Be0512301D0B19C3C0A9300bAA1E03758", 
+            "sellAmount": "1000000", 
+            "buyAmount": "1000000", 
+            "validTo": 1639052668, 
+            "appData": "0x000000000000000000000000000000000000000000000000000000000000ca1f", 
+            "feeAmount": "887",
+            "kind": "buy", 
+            "partiallyFillable": false, 
+            "sellTokenBalance": "erc20", 
+            "buyTokenBalance": "erc20", 
+            "signature": "0x547b936d1aac1b0153776887ee23a92097d3988c433b0d9642014c9861e07f8f3e16cbc7d1c2d42d3f3b32b332f2aafc86daace610a0a5968b93733c79c87a1a1b", 
+            "signingScheme": "eip712"}""")
+        self.assertEqual(order.set_json, expected_order)
```

Comment:
Does this actually work without the brackets here?
```suggestion
        self.assertEqual(order.set_json(), expected_order)
```